conftest.py

```diff
@@ -0,0 +1,3 @@
+import pytest
+
+pytest.register_assert_rewrite("datatree.testing")
```

Comment:
I don't really understand why [I needed this in order to display messages](https://stackoverflow.com/questions/54796599/python-pytest-does-not-show-assertion-differences) from `assert` statements, but xarray doesn't seem to ever call this...